packages/frontend/src/components/common/Table/ScrollableTable/TableBody.tsx

```diff
@@ -16,20 +19,28 @@ const TableBody = () => {
                         const meta = cell.column.columnDef
                             .meta as TableColumn['meta'];
                         return (
-                            <BodyCell
-                                key={cell.id}
-                                $rowIndex={rowIndex}
-                                $isNaN={
-                                    !meta?.item || !isNumericItem(meta.item)
-                                }
+                            <CellContextMenu
+                                cell={cell}
+                                onOpen={() => setSelectedCell(cell.id)}
+                                onClose={() => setSelectedCell(undefined)}
                             >
-                                <CellContextMenu cell={cell}>
-                                    {flexRender(
-                                        cell.column.columnDef.cell,
-                                        cell.getContext(),
-                                    )}
-                                </CellContextMenu>
-                            </BodyCell>
+                                <BodyCell
+                                    key={cell.id}
```

Comment:
Should we move the `key` to the `CellContextMenu` ?

Reply:
good catch